_posts/2019-06-25-lambda-quickstart.md

```diff
@@ -0,0 +1,320 @@
+            "Effect": "Allow",
+            "Action": "s3:*Object",
+            "Resource": [
+                "arn:aws:s3:::kubecon-2019/*",
+                "arn:aws:s3:::containerdays-2019/*"
+            ]
+        }
+    ]
+}
+``` 
+
+- Create a [new IAM policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html#access_policies_create-start), with the above minimum permissions. For example call this new policy `OpenFaaS Lambda Function Policy`.
+- Create a [new IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user.html), and assign it the above police. For example call this new role `OpenFaaS Lambda Function Role`. 
+
+OR
+
+##### 2.3.2. No access function IAM Role
+
+```javascript
+{
+    "Version": "2012-10-17",
+    "Statement": [
+       {
+           "Sid": "CreateLogs",
+           "Effect": "Allow",
+           "Action": [
+               "logs:CreateLogGroup",
+               "logs:CreateLogStream",
+               "logs:PutLogEvents"
+           ],
+           "Resource": "*"
+       }
+    ]
+}
+``` 
+
+- Create a [new IAM policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html#access_policies_create-start), with the above minimum permissions. For example call this new policy `OpenFaaS Lambda Function Policy`.
+- Create a [new IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user.html), and assign it the above policy. For example call this new role `OpenFaaS Lambda Function Role`. 
+
+#### 2.4. Assign AWS access credentials to faas-lambda using Kubernetes secrets
+This is a secure way to pass the AWS access credentials to the `faas-lambda` provider
+
+```
+$ export AWS_ACCESS_KEY_ID={value from step}
+export AWS_SECRET_ACCESS_KEY={value from last step}
+
+kubectl -n openfaas create secret generic lambda-aws-access \
+   --from-literal=AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
+   --from-literal=AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+```
+
+#### 2.5. Deploy OpenFaaS helm chart
+
+
+##### 2.5.1 Gather Helm configuration arguments
+
+| Parameter                        | Description |
+| -------------------------------- | ----------- |
+| aws_region                       | AWS region that your Lambda functions will run in. For example `eu-west-1` |
+| faaslambda.lambda_execution_role | IAM role to assign each function. See step 2.3. for more information   . |
+| access_token                     | Early access token obtained from step 1. |
+| access_email                     | Email address used when registering token from step 1. |
+
+
+##### 2.5.2 Deploy chart
+
+Below are the commands to clone and deploy the official OpenFaaS Helm chart, configured to deploy to AWS Lambda.
+Please change the `--set` parameters `faaslambda.aws_region`, `faaslambda.lambda_execution_role`, `faaslambda.access_token` and
+`faaslambda.access_email` with the values you gathered together in the previous step.
+
+```bash
```

Comment:
Why are we cloning the repo?

I'm fairly sure that `access_token` should also be a Kubernetes secret, this looks a bit inconsistent/leaky.